### ISSUE TITLE
Verify namespace is scheduled every day (in the first month of each quarter)

### ISSUE THREAD

**Issue by domsj** [openvstorage/framework-alba-plugin#252]
I noticed a lot of verify namespace work items on some env, seems like (this month) they are scheduled every day.
Verify namespace is quite costly and couldn't finish it's work in a day so maintenance barely had time/resources to do the other stuff it's supposed to do.

```
ctrl-stras-03 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.2.gz:2016-10-25 02:00:00 35300 +0200 - ctrl-stras-03 - 105710/140362678363968 - lib/scheduled tasks - 154980 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.2.gz:2016-10-25 02:03:36 54000 +0200 - ctrl-stras-03 - 105710/140362678363968 - lib/scheduled tasks - 156382 - INFO - verify namespace task scheduling finished

ctrl-grav-02 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.6.gz:2016-10-21 02:00:00 07400 +0200 - ctrl-grav-02 - 38759/140231241877312 - lib/scheduled tasks - 19723 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.6.gz:2016-10-21 02:03:23 58200 +0200 - ctrl-grav-02 - 38759/140231241877312 - lib/scheduled tasks - 21125 - INFO - verify namespace task scheduling finished

ctrl-roub-01 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.4.gz:2016-10-23 02:00:00 14100 +0200 - ctrl-roub-01 - 19071/140011497117504 - lib/scheduled tasks - 132458 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.4.gz:2016-10-23 02:02:04 14000 +0200 - ctrl-roub-01 - 19071/140011497117504 - lib/scheduled tasks - 133858 - INFO - verify namespace task scheduling finished

perf-roub-04 | FAILED | rc=1 >>


perf-roub-03 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.1.gz:2016-10-26 02:00:00 04300 +0200 - perf-roub-03 - 33262/139771296581440 - lib/scheduled tasks - 731 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.1.gz:2016-10-26 02:02:21 49300 +0200 - perf-roub-03 - 33262/139771296581440 - lib/scheduled tasks - 2129 - INFO - verify namespace task scheduling finished

perf-roub-02 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.3.gz:2016-10-24 02:00:00 20200 +0200 - perf-roub-02 - 12174/139875046467392 - lib/scheduled tasks - 150135 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.3.gz:2016-10-24 02:02:45 27700 +0200 - perf-roub-02 - 12174/139875046467392 - lib/scheduled tasks - 151533 - INFO - verify namespace task scheduling finished
/var/log/upstart/ovs-workers.log.5.gz:2016-10-22 02:00:00 18200 +0200 - perf-roub-02 - 65231/139875046467392 - lib/scheduled tasks - 131423 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.5.gz:2016-10-22 02:02:20 42300 +0200 - perf-roub-02 - 65231/139875046467392 - lib/scheduled tasks - 132833 - INFO - verify namespace task scheduling finished

perf-roub-01 | FAILED | rc=1 >>


cap-roub-01 | FAILED | rc=1 >>


cap-roub-02 | FAILED | rc=1 >>


perf-stras-06 | FAILED | rc=1 >>
/var/log/upstart/ovs-workers.log.7.gz:2016-10-20 02:00:00 11800 +0200 - perf-stras-06 - 18472/139812738627392 - lib/scheduled tasks - 48112 - INFO - verify namespace task scheduling started
/var/log/upstart/ovs-workers.log.7.gz:2016-10-20 02:04:18 77300 +0200 - perf-stras-06 - 18472/139812738627392 - lib/scheduled tasks - 49526 - INFO - verify namespace task scheduling finished
```

**Comment by khenderick** [comment]
Fixed by #254, packaged in openvstorage-2.7.4-rev.4204.f4c4cb8

**Comment by JeffreyDevloo** [comment]
### Information
Referring to the celery scheduling docs:
```
"It is important to realize that any day on which execution should occur must be represented by entries in all three of the day and month attributes.
For example, if day_of_week is 0 and day_of_month is every seventh day, only months that begin on Sunday and are also in the month_of_year attribute will have execution events. 
Or, day_of_week is 1 and day_of_month is ‘1-7,15-21’ means every first and third monday of every month present in month_of_year."
```
Previously the task did not specify any days.
```
@celery.task(name='alba.scheduled.verify_namespaces', schedule=Schedule(minute='0', hour='0', month_of_year='*/3'))
```
Meaning that it defaulted to days=* which meant everyday of the first month of each quarter.
Currently the task is scheduled on the first day on 24:00 on the first month of each quarter.
```
@celery.task(name='alba.scheduled.verify_namespaces', schedule=Schedule(minute='0', hour='0', day_of_month='1', month_of_year='*/3')
```

### Test result
Test passed.
